team_builder_react/src/components/Results.js

```diff
@@ -6,10 +6,71 @@ import Container from 'react-bootstrap/Container';
 import Navbar from 'react-bootstrap/Navbar';
 import Stack from 'react-bootstrap/Stack';
 import Row from 'react-bootstrap/Row';
-import Col from 'react-bootstrap/Col';
+import Alert from 'react-bootstrap/Alert';
+import { GetEmployeeList, GetValidTeam, SendTeamParameters } from '../utilities/API_Interface.js';
 
-const Results = () => {
+const Results = ({techList, num}) => {
     const labels = ['Confidence', 'Delegator', 'Determination', 'Selling', 'Relationship', 'Disrupter', 'Knowledge', 'Independance', 'Profitability', 'Risk'];
+    const [names, setNames] = useState([])
+    const [scores, setScores] = useState([])
+    const [teamResponse, setTeamResponse] = useState(false)
+    const [team, setTeam] = useState(false)
+    const [firstTime, setFirstTime] = useState(false)
+    loadData();
+
+    function loadData(){
+
+      if (!firstTime ){
+        setFirstTime(true)
+        
+        var id = SendTeamParameters(num, techList)
+        .then((id) => {
+            console.log("Response ID: " + id);
+
+            GetValidTeam(id)
+            .then((response) => {
+                console.log("Team Response: " + response);
+
+                // GetEmployeeList(response.data)
+                GetEmployeeList([1, 2, 3])
+                .then((team) => {
+                    console.log("Response for getting employees:");
+                    console.log(team);
+                    setTeam(team);
+                    var temp = []
+                    var scoresTemp = [0,0,0,0,0,0,0,0,0,0]
+
+                    for (const member of team) {
+                      temp.push(`${member[0].name_last}, ${member[0].name_first}`);
+
+                      scoresTemp[0] += member[0].bpt_confidence
+                      scoresTemp[1] += member[0].bpt_delegator
+                      scoresTemp[2] += member[0].bpt_determination
+                      scoresTemp[3] += member[0].bpt_disruptor
+                      scoresTemp[4] += member[0].bpt_independence
+                      scoresTemp[5] += member[0].bpt_knowledge
+                      scoresTemp[6] += member[0].bpt_profitability
+                      scoresTemp[7] += member[0].bpt_relationship
+                      scoresTemp[8] += member[0].bpt_risk
+                      scoresTemp[9] += member[0].bpt_selling
```

Comment:
Maybe make scoreTemp an object where the key is the trait and the value is the score? It would just make it more readable and less arbitrary (so indexes don't have to map to a trait).